src/Compilers/CSharp/Portable/Compilation/MemberSemanticModel.cs

```diff
@@ -959,29 +959,74 @@ public override SyntaxTree SyntaxTree
+            if (parameter != null && parameter.Default?.FullSpan.Contains(node.Span) == true)
+            {
+                bindingRoot = parameter.Default;
+            }
+
+            // if binding root is field variable declarator, make it initializer
+            // we need to do this since node map doesn't contain bound node for field/event variable declarator
+            var variableDeclarator = bindingRoot as VariableDeclaratorSyntax;
+            if (variableDeclarator != null && variableDeclarator.Initializer?.FullSpan.Contains(node.Span) == true)
+            {
+                if (variableDeclarator.Parent?.Parent.IsKind(SyntaxKind.FieldDeclaration) == true ||
+                    variableDeclarator.Parent?.Parent.IsKind(SyntaxKind.EventFieldDeclaration) == true)
+                {
+                    bindingRoot = variableDeclarator.Initializer;
+                }
+            }
+
+            // if binding root is enum member decleration, make it equal value
+            // we need to do this since node map doesn't contain bound node for enum member decl
+            var enumMember = bindingRoot as EnumMemberDeclarationSyntax;
+            if (enumMember != null && enumMember.EqualsValue?.FullSpan.Contains(node.Span) == true)
+            {
+                bindingRoot = enumMember.EqualsValue;
+            }
+
+            // if binding root is property member decleration, make it equal value
+            // we need to do this since node map doesn't contain bound node for property initializer
+            var propertyMember = bindingRoot as PropertyDeclarationSyntax;
+            if (propertyMember != null && propertyMember.Initializer?.FullSpan.Contains(node.Span) == true)
+            {
+                bindingRoot = propertyMember.Initializer;
+            }
+
+            return bindingRoot;
+        }
+
+        private IOperation GetStatementOrRootOperation(CSharpSyntaxNode node, CancellationToken cancellationToken)
+        {
+            Debug.Assert(node == GetBindingRootOrInitializer(node));
+
             CSharpSyntaxNode bindableNode;
 
             BoundNode lowestBoundNode;
             BoundNode highestBoundNode;
-            BoundNode boundParent;
+            GetBoundNodes(node, out bindableNode, out lowestBoundNode, out highestBoundNode, out _);
 
-            GetBoundNodes(node, out bindableNode, out lowestBoundNode, out highestBoundNode, out boundParent);
-            BoundNode result;
-            switch (options)
-            {
-                case GetOperationOptions.Parent:
-                    result = boundParent;
-                    break;
-                case GetOperationOptions.Highest:
-                    result = highestBoundNode;
-                    break;
-                case GetOperationOptions.Lowest:
-                default:
-                    result = lowestBoundNode;
-                    break;
-            }
+            BoundNode result = highestBoundNode ?? lowestBoundNode;
```

Comment:
>highestBoundNode [](start = 31, length = 16)

Why would we ever want the highestBoundNode if it is different from the lowestBoundNode?